test/residency/prototypes/test_task017_llamacpp_soft_release_public_seam.py

```
@@ -0,0 +1,1757 @@
+        *fallback_output_rows(),
+        f"platform.current={platform_id}",
+        "runtime_authority=none",
+    ]
+
+
+def fail_unavailable() -> None:
+    print(FAILURE, file=sys.stderr)
+    raise SystemExit(1)
+
+
+def require(value: bool, message: str) -> None:
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)